storage/boltdb/client.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/boltdb/bolt"
 	"github.com/zeebo/errs"
-	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+	"gopkg.in/spacemonkeygo/monkit.v2"
```

Comment:
`monkit` should not be removed

Reply:
Idk what happened here, ill put it back

Reply:
I'm not sure we need that alias for the package?